data/korath/korath missions.txt

```diff
@@ -257,3 +257,331 @@ mission "Small Scale Delivery"
+			`	Your communicator notifies you of a payment of <payment>.`
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti"
+		government "Kor Mereti"
+		to spawn
+			random < 30
+		personality
+			staying
+
+
+mission "Efreti: Distress Call 3"
+	name "Recover the fallen crew of the <EDC-ship>"
+	description "Recover the bodies of the dead <EDC-ship> crew on <destination> and return them to <origin>."
+	source "Laki Nemparu"
+	destination "Seleptra Nak"
+	cargo "Efreti coffins" 13
+	landing
+
+	to offer
+		has "Efreti: Distress Call 2: done"
+
+	on offer
+		conversation
+			`A few hours later, you are relaxing at a spaceport restaurant now that the evacuation is complete when a delegation of Korath approach you somewhat hesitantly. They are dressed in what appears to be much finer clothing than most Korath you have met, certainly nothing you would wear to do work in a spaceport. As they make the traditional gesture of greeting, which you return, you are curious both as to what they want and how they will ask.`
+			`	One of them takes a small case out of a pocket and removes a black device that looks like three triangles hooked together. He hisses at it for a moment and it rises out of his hand. The air around it begins to shimmer, followed by the appearance of a life-sized hologram of a Quarg.`
+			`	"Salutations, far traveler," the Quarg rumbles. "These Korath-Friends are the princes of the clan that you assisted. While grateful for the living, they cannot fully mourn those that lie dead. They beseech you travel back to <planet> and bring their lost ones home to <origin>."`
+			choice
+				`	"I will bring them home."`
+					goto home
+				`	"I will if they pay me."`
+					goto pay
+				`	"I won't risk my ship for the dead."`
+			`	The leader of the group visibly sags as the Quarg translates your words. Two younger Korath bear him up as the group turns and leaves.`
+				decline
+
+			label home
+			`	You hadn't realized how tense the group was until you saw them relax as your words reach them. They seem relieved at your answer.`
+				goto title
+
+			label pay
+			`	The group relaxes quickly as you see a ripple of relief and pride pass through them.`
+
+			label title
+			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "Efreti crews have families, as children learn by working with parents. For some families, three generations were lost. We will not try to salvage the ship, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
+				accept
+	npc
+		system "Seketra"
+		fleet "Small Kor Mereti"
+		government "Kor Mereti"
+		to spawn
+			random < 30
+		personality
+			staying
+
+mission "Efreti: Distress Call 4"
+	landing
+	name "Recover the fallen of <EDC-ship>"
+	description "Return the bodies of the dead <EDC-ship> crew to <planet>."
+	source "Seleptra Nak"
+	destination "Karek Fornati"
+	cargo "Efreti deceased" 13
```

Comment:
Efreti is a proper noun and should be capitalized, regardless of errors elsewhere in Endless Sky txt.

Reply:
While that is true, it is intentional we don't use capital words even if proper for data such as cargo lines. It's not a matter of the word being proper, it's a matter of how we format the data, in which this case it should be lower case.

Reply:
It isn't merely data if it is displayed to the user.

Reply:
Regardless, if the convention is done the same way everywhere else, please do so in this PR. As you have said, if it should be fixed, please do so in another PR, and keep to the standard in this one, rather than make it confusing here.

Reply:
Oh! I see now. No matter how you capitalize it, the code automatically displays it capitalized in-game:

![Screenshot_2023-05-27_13-28-14](https://github.com/endless-sky/endless-sky/assets/116329264/f7dfd0e7-2415-427f-a2cc-e34d153de713)